### PR TITLE
Update to API version v1

### PR DESCRIPTION
**Pull request by spadgett**
@liggitt @jwforres

### CODE REVIEW

File: views/replication-controller.html [hunk 1]
```diff
@@ -8,8 +8,7 @@
     <dt>Created</dt>
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
     <dt>Replicas</dt>
-    <!-- TODO: Update for API version v1. The default has changed from 0 to 1. -->
-    <dd>{{resource.spec.replicas || 0}}</dd>
+    <dd>{{(resource.spec.replicas === undefined) ? 1 : resource.spec.replicas}}</dd>
```
Comment (liggitt) [inline views/replication-controller.html:11]: did you make sure angular evaluates this correctly?

File: fixtures/pod.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 window.EXAMPLE_POD = {
   "kind": "Pod",
-  "apiVersion": "v1beta3",
```
Comment (liggitt) [inline fixtures/pod.js:3]: add `serviceAccountName` to the pod fixture and to the pod view

**Comment by liggitt** [comment]
looks pretty good to me. note to self/@jwforres: before merging this, create a branch from master for v1beta3. when cutting a release for this, increment a major version to 1.0.0

File: views/replication-controller.html (hunk 1 above)
Comment (spadgett) [inline views/replication-controller.html:11]: When testing, I removed `replicas` from the fixture, and it worked.

File: fixtures/replicationController.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 window.EXAMPLE_RC = {
   "kind": "ReplicationController",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
```
Comment (liggitt) [inline fixtures/replicationController.js:3]: add `serviceAccountName` to the rc spec.template and to the replication controller view

**Comment by spadgett** [comment]
kubernetes_describer_example

File: views/replication-controller.html [hunk 4]
```diff
@@ -8,8 +8,9 @@
...
+    <dt ng-if="resource.spec.serviceAccountName">Service account</dt>
+    <dd ng-if="resource.spec.serviceAccountName">{{resource.spec.serviceAccountName}}</dd>
```
Comment (liggitt) [inline views/replication-controller.html]: serviceAccountName is part of the pod spec inside the rc spec (rc.spec.template.spec.serviceAccountName)

Comment (liggitt) [inline views/replication-controller.html]: update the fixture, too

Comment (liggitt) [inline views/replication-controller.html]: in general, double-check fixtures against reality whenever possible :)

**Comment by spadgett** [comment]
@liggitt I was looking at `pkg/api/graph/test/bare-rc.yaml`, which has a service account, but misread it... :(

**Comment by stefwalter** [comment]
I've tested this in Cockpit roughly and it works. We updated our core code to the v1 API a few days ago and the object describer was one of the things that is remaining. CC @sub-mod

**Comment by liggitt** [comment]
LGTM

**Pull request merged**